tasks/cephfs/test_volume_client.py

```diff
@@ -64,6 +64,58 @@ def _configure_vc_auth(self, mount, id_name):
         self._sudo_write_file(mount.client_remote, mount.get_keyring_path(), out)
         self.set_conf("client.{name}".format(name=id_name), "keyring", mount.get_keyring_path())
 
+    def _configure_guest_auth(self, volumeclient_mount, guest_mount,
+                              guest_entity, group_id, volume_id,
+                              readonly=False):
```

Comment:
When you rebase you'll need to optionally pass through the volume_prefix and namespace_prefix for use with test_lifecycle, and pass it into volume_client_python